index.js

```diff
@@ -80,7 +80,10 @@ Electron.prototype._spawn = function(url){
   debug('spawn %s', url);
 
   var self = this;
-  var ps = self.ps = spawn(electron, [runner], {
+  var electronOpts = []
+  if (!self.opts.sandbox) electronOpts.push('--no-sandbox')
```

Comment:
Having the sandbox turned on should be the default, not an opt-in, for security reasons

Reply:
Ok, I agree completely with the default of using a sandbox. I think the best place to ensure that is to add `this.sandbox = 'sandbox' in this.opts ? this.opts.sandbox : true` [within the Electron(opts) function](https://github.com/juliangruber/electron-stream/blob/db2902fd4d7c997c1dc709eaf496debfb005e2f6/index.js#L24-L27.). I can then change line 84 to `if (!self.sandbox) electronOpts.push('--no-sandbox')`. 

I will submit a pull request with the above changes, but if you disagree with the fix, just let me know.

In my local `browser-run` and `tape-run` clones, using `optimist` I did specify 
```javascript
 .describe('sandbox', 'Use "--no-sandbox" to run electron outside of a sandbox, may be required in a docker container to pipe browser stdout and stderr streams to another process or console')
  .default('sandbox', true)
```

but obviously I had failed to account for electron-stream usage outside of browser-run and tape-run, hence the additional fixes are needed as you noted.